test/integration/Proposal.test.ts

```diff
@@ -192,8 +192,8 @@ describe('Proposal', () => {
     test('Should search Proposals open after block 1000', async () => {
```

Comment:
multiple tests until here are still using blockStart and blockEnd

Reply:
testData is broken:
     const testData = {
         submitter: 'pasdfasfsdfad',
         blockStart: 1000,
         blockEnd: 1010,
         type: ProposalType.PUBLIC_VOTE,
         title:  'proposal x title',
         description: 'proposal to x',
         expiryTime: 4,
         postedAt: new Date().getTime(),
         expiredAt: new Date().getTime() + 100000000,
         receivedAt: new Date().getTime()
     } as ProposalCreateRequest;

replace blockStart, blockEnd with timeStart, timeEnd

all times are stored as ms after 1970 in the db.

Reply:
Tests fixed